extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ConfigSourceName.java

```diff
@@ -0,0 +1,139 @@
+package io.quarkus.vertx.http.runtime.devmode;
+
+import java.util.Objects;
+
+public class ConfigSourceName implements Comparable<ConfigSourceName> {
+    private static final String APPLICATION_PROPERTIES_CONFIG_SOURCE = "PropertiesConfigSource[source=application.properties]";
+    private static final String SYSTEM_PROPERTIES_CONFIG_SOURCE = "SysPropConfigSource";
+    private static final String ENVIRONMENT_CONFIG_SOURCE = "EnvConfigSource";
+    private static final String PROPERTIES_CONFIG_SOURCE = "PropertiesConfigSource";
+    private static final String DEFAULT_VALUES = "default values";
```

Comment:
```suggestion
    private static final String DEFAULT_VALUES = "Default values";
```
would be better given it's the title of a section.

Reply:
It's called `default values` in config, but I can change the display name to be `Default values`. That is what you want right ?

Reply:
(So the config source name is `default values`)

Reply:
Done. Will now show on the screen as `Default value`.